src/components/css-components/Alert/alert.css

```diff
@@ -0,0 +1,92 @@
+.comp-preview .alert {
+  margin: 4px 4px;
+}
+
+.alert {
+  padding: 15px;
```

Comment:
why are you not using rem instead of pixels as it helps in responsive design.

Reply:
Ya correct! 
Somehow missed at some places. Will fix it now. 
Thanks for pointing it out.